db/migrate/20220619021917_add_created_by_user_to_announcement.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatedByUserToAnnouncement < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :announcements, :user, null: false, foreign_key: true
```

Comment:
We'll have issues with the null false and the existing announcements. But we can do the same as we did the last time by removing the null false, merging, updating the existing announcements to be created by one of our accounts and then re-add the null false constraint.

Reply:
Any thoughts on what to do with the historical ones? Is there a generic super user admin account they can be tied to?

Reply:
There isn't any generic super users unfortunately, we could create one or associate them with one of the existing admin accounts for now if that's easier? I can sort this out through the Heroku console if you want?

Reply:
Open to ideas, but I think probably seems like the simplest solution outside of making the relationship optional and not forcing the null constraint. 

Probably better to tie them to a generic account just for the sake of accuracy, but I guess it depends on how many actual records there are, is it worth the effort of tying existing ones to the right person?